TCAT/Network/Models.swift

```diff
@@ -93,76 +53,13 @@ class RouteSectionsObject: Codable {
     var walking: [Route]
 }
 
-struct MultiRoutesRequest: Codable {
-    var success: Bool
-    var data: [Route?]
+internal struct GetDelayBody: Codable {
+    let stopId: String
+    let tripId: String
 }
 
-class AllBusStopsRequest: Codable {
+// Response
+struct Response<T: Codable>: Codable {
     var success: Bool
-    var data: [Place]
-
-    private enum CodingKeys: CodingKey {
-        case success
-        case data
-    }
-
-    required init(from decoder: Decoder) throws {
-        let container = try decoder.container(keyedBy: CodingKeys.self)
-        success = try container.decode(Bool.self, forKey: .success)
-        data = try container.decode([Place].self, forKey: .data)
-        parseAllStops()
-    }
-
-    // TO BE MOVED TO BACKEND
```

Comment:
This is confirmed moved to backend?

Reply:
Never mind, I see it was just moved. Should get in backend!

Reply:
Talked to kevin about this. He said he'll do it soon

Reply:
@kevinchan159 Do you mind making an issue? Omar or I can clarify anything